src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -183,66 +176,58 @@ function addScriptTagsToDocument(resourceLoader: WebAssemblyResourceLoader) {
     .keys(resourceLoader.bootConfig.resources.runtime)
     .filter(n => n.startsWith('dotnet.') && n.endsWith('.js'))[0];
   const dotnetJsContentHash = resourceLoader.bootConfig.resources.runtime[dotnetJsResourceName];
-  const scriptElem = document.createElement('script');
-  scriptElem.src = `_framework/${dotnetJsResourceName}`;
-  scriptElem.defer = true;
-
-  // For consistency with WebAssemblyResourceLoader, we only enforce SRI if caching is allowed
-  if (resourceLoader.bootConfig.cacheBootResources) {
-    scriptElem.integrity = dotnetJsContentHash;
-    scriptElem.crossOrigin = 'anonymous';
-  }
+  let src = `./${dotnetJsResourceName}`;
+
 
   // Allow overriding the URI from which the dotnet.*.js file is loaded
   if (resourceLoader.startOptions.loadBootResource) {
     const resourceType: WebAssemblyBootResourceType = 'dotnetjs';
-    const customSrc = resourceLoader.startOptions.loadBootResource(resourceType, dotnetJsResourceName, scriptElem.src, dotnetJsContentHash);
+    const customSrc = resourceLoader.startOptions.loadBootResource(resourceType, dotnetJsResourceName, src, dotnetJsContentHash);
     if (typeof (customSrc) === 'string') {
-      scriptElem.src = customSrc;
+      src = customSrc;
```

Comment:
Are you sure this works? It looks like it would be a breaking change. If an existing app returns a URL that doesn't start with `./`, then it won't work with `import`.

Reply:
Changed it to `new URL(`_framework/${dotnetJsResourceName}`, document.baseURI)` hope that's better